### PR TITLE
input-remapper: Initial inclusion

### LINKED ISSUE #1275: Input Remapper
### Homepage

https://github.com/sezanzeb/input-remapper/releases

### Why should this be included in the repository?

This is useful for gamers or people that use custom hardware that requires a special UI to map keys, adding this to the repo would be greatly appreciated by many as it would allow them to customize the way they use their hardware.

### Are we allowed to redistribute it?

GPL-3.0 license

### What kind of user will use this package, and how many users do you think will use this package?

Tinkerers, Gamers and people with custom hardware that requires mapping.

### Link to source archive file

https://github.com/sezanzeb/input-remapper/releases/tag/2.0.1

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**

Resolves #1275.

**Test Plan**

Remap my F13 key to KEY_DEL

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by joebonrichie** [comment]
Only thing i'm not particularly happy about here is that pydbus hasn't seen a release in quite a while. However, python-dbus is what most python projects use and that is even more unmaintained so xD

File: packages/i/input-remapper/pspec_x86_64.xml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    </Source>
+    <Package>
+        <Name>input-remapper</Name>
+        <Summary xml:lang="en">An easy to use tool to change the behaviour of your input devices.</Summary>
+        <Description xml:lang="en">An easy to use tool to change the behaviour of your input devices. Supports X11, Wayland, combinations, programmable macros, joysticks, wheels, triggers, keys, mouse-movements and more. Maps any input to any other input.
+</Description>
+        <PartOf>system.utils</PartOf>
+        <Files>
+            <Path fileType="config">/etc/dbus-1/system.d/inputremapper.Control.conf</Path>
+            <Path fileType="config">/etc/xdg/autostart/input-remapper-autoload.desktop</Path>
```
Comment (EbonJaeger) [inline packages/i/input-remapper/pspec_x86_64.xml:24]: Are these the correct directories? XDG autostart at least I'm sure has a stateless directory.

Comment (joebonrichie) [reply packages/i/input-remapper/pspec_x86_64.xml:24]: oh you're right, thanks

**Review by ReillyBrogan — APPROVED** [review]
LGTM, Thanks!

**Comment by Napyr** [comment]
I love you guys.

**Pull request merged**